lib/screen_controller.dart

```diff
@@ -720,8 +720,12 @@ class ScreenController {
     Map<String, dynamic>? pageArgs,
   }) {
     PageType pageType = asModal == true ? PageType.modal : PageType.regular;
-
-    Widget screenWidget =
+    Widget? screenWidget;
+    if (Ensemble().customFlutterWidgets?.containsKey(screenName) ?? false) {
```

Comment:
This will just take the user to Page Not Found error when running in Preview. Instead we want to say something like "External Screen Placeholder", and include a link to an Ensemble Doc's page where we explain how to use it with ensemble_starter to get your page to appear.

This means we probably want the user to explicitly say `external: true` when using navigateScreen*. This way you just check for the flag presence if not found, then show that message

Reply:
make sense, we'll do the changes